plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergTypes.java

```diff
@@ -76,8 +77,7 @@ public static Object convertIcebergValueToTrino(Type icebergType, Object value)
         }
         if (icebergType instanceof Types.BinaryType) {
             // TODO the client sees the bytearray's tostring ouput instead of seeing actual bytes, needs to be fixed.
-            // TODO return Slice
-            return ((ByteBuffer) value).array().clone();
+            return Slices.wrappedBuffer(((ByteBuffer) value).array().clone());
```

Comment:
remove `// TODO the client sees the bytearray's tostring ouput instead of se` too